tests/v2/cassettes/test_scenarios/test_delete_an_existing_incident_returns_ok_response.yaml

```diff
@@ -23,13 +23,11 @@ interactions:
       accept:
       - application/json
     method: DELETE
-    uri: https://api.datadoghq.com/api/v2/incidents/3002d31f-1c9c-56b5-ae84-8363d9c3ee05
+    uri: https://api.datadoghq.com/api/v2/incidents/50ec6462-78d8-5efa-962a-3c120a199b1a
   response:
     body:
       string: ''
-    headers:
-      content-type:
-      - application/json
+    headers: {}
```

Comment:
@therve are we dropping all response headers in purpose?

Reply:
Yeah this is the effect of https://github.com/DataDog/datadog-api-spec/pull/1394, just we didn't re-record those as the tests were skipped.